src/stream-transformer.ts

```diff
@@ -78,14 +78,15 @@ export function createOpenAIStreamTransformer(model: string): TransformStream<St
 						{
 							index: 0,
 							delta: delta,
-							finish_reason: null,
-							logprobs: null,
-							matched_stop: null
+							finish_reason: null
 						}
-					],
-					usage: null
+					]
 				};
-				controller.enqueue(encoder.encode(`data: ${JSON.stringify(openAIChunk)}\n\n`));
+				controller.enqueue(
+					encoder.encode(`data: ${JSON.stringify(openAIChunk)}
+
+`)
```

Comment:
The formatting looks off, because of the enter. Is this intented? Because we had \n before